.github/codeql.yml

```
@@ -2,6 +2,9 @@
 
 name: Shescape CodeQL config
 
+packs:
+  - githubsecuritylab/codeql-javascript-queries
```

```suggestion
  - githubsecuritylab/codeql-javascript-queries@0.0.4
```

See https://github.com/GitHubSecurityLab/CodeQL-Community-Packs/pkgs/container/codeql-javascript-queries

---

Resolved with d298d3390046b96969d65a02c372c752f165c4ba